tools/cbmc/proofs/ARP/ARP_FreeRTOS_OutputARPRequest/OutputARPRequest_harness.c

```diff
@@ -24,7 +52,18 @@ NetworkBufferDescriptor_t xNetworkBuffer;
  * of an ARPPacket to gurantee memory safety.
  */
 NetworkBufferDescriptor_t *pxGetNetworkBufferWithDescriptor( size_t xRequestedSizeBytes, TickType_t xBlockTimeTicks ){
-	xNetworkBuffer.pucEthernetBuffer = malloc(xRequestedSizeBytes);
+	#ifdef CBMC_PROOF_ASSUMPTION_HOLDS
```

Comment:
I had heretofore been assuming that our CI will compile and run each proof once. Is that true? If not, and we can get coverage of these various static configurations, great.